packages/core/.gitignore

```diff
@@ -1,5 +1,6 @@
 dist/
 .ponder/
+ponder-env.d.ts
```

Comment:
I wonder if we should mock the `runCodegen` function in `run.test.ts` and just assert that it gets called.